debian/securedrop-workstation-config.install

```diff
@@ -1,6 +1,7 @@
 workstation-config/mailcap.default opt/sdw/
 workstation-config/mimeapps.list.sd-viewer opt/sdw/
 workstation-config/mimeapps.list.sd-app opt/sdw/
-workstation-config/mimeapps.list.sd-devices-dvm opt/sdw/
+workstation-config/mimeapps.list.sd-devices opt/sdw/
 workstation-config/open-in-dvm.desktop opt/sdw/
 workstation-config/paxctld.conf opt/sdw/
+workstation-config/securedrop-mime-handling usr/sbin/
```

Comment:
sbin is in theory supposed to be stuff only run by root, but this is being run by `user`. I think we'd be better putting it in normal `/usr/bin/`.